### PR TITLE
[Teswiz-upgrade] Documentation update for teswiz version v0.0.83

### PR DESCRIPTION
**Pull request by snehith07**
Updated Readme file and documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,113 @@
 [![CodeQL](https://github.com/znsio/teswiz/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/znsio/teswiz/actions/workflows/codeql-analysis.yml)
 
 # To Build
+
 `./gradlew clean build`
 
+# ![#f03c15](https://placehold.co/15x15/f03c15/f03c15.png) Breaking changes in Latest teswiz v0.0.83![#f03c15](https://placehold.co/15x15/f03c15/f03c15.png)
```
Comment (anandbagmar) [inline README.md:10]: This has been moved to the new docs/BreakingChanges-README.md file. We do not want to get it back here.

File: README.md [hunk 2]
```diff
@@ -18,14 +123,16 @@ them against
 * Windows Apps
 * Web
 
-Applitools (https://applitools.com/) Visual AI, and Applitools Ultrafast Grid (https://applitools.com/product-ultrafast-test-cloud/) is integrated with this framework, to provide
+Applitools (https://applitools.com/) Visual AI, and Applitools Ultrafast
```
Comment (anandbagmar) [inline README.md:126]: no need for a line break

File: README.md [hunk 3]
```diff
@@ -18,14 +123,16 @@ them against
...
+Grid (https://applitools.com/product-ultrafast-test-cloud/) is integrated with this framework, to provide
 Visual AI testing as part of functional automation.
 
 Reports will be uploaded to reportportal.io, that you would need to setup separately, and provide the server details in
 src/test/resources/reportportal.properties file or provide the path to the file using this environment
 variable: `REPORT_PORTAL_FILE`
 
-Test can run on local browsers / devices, or against any cloud provider, such as HeadSpin, BrowserStack, SauceLabs, pCloudy.
+Test can run on local browsers / devices, or against any cloud provider, such as HeadSpin, BrowserStack, SauceLabs,
+pCloudy.
```
Comment (anandbagmar) [inline README.md:135]: no need for a line break

File: README.md (hunk 1 above)
Comment (snehith07) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (snehith07) [reply README.md]: updated

File: README.md (hunk 3 above)
Comment (snehith07) [reply README.md]: updated

**Pull request merged**